packages/client/api/src/util.ts

```diff
@@ -0,0 +1,32 @@
+import type { IFluenceClient } from '@fluencelabs/interfaces';
+
+const getPeerFromGlobalThis = (): IFluenceClient | undefined => {
+    // @ts-ignore
+    return globalThis.defaultPeer;
+};
+
+// TODO: DXJ-271
+const REJECT_MESSAGE = 'You probably forgot to add script tag. Read about it here: ';
+
+/**
+ * Wait until the js client script it loaded and return the default peer from globalThis
+ */
+export const getDefaultPeer = (): Promise<IFluenceClient> => {
+    return new Promise((resolve, reject) => {
+        let interval: NodeJS.Timer | undefined;
```

Comment:
if it's for frontend that would be a number not NodeJS.Timer. Doesn't reallty affect anything right here but it's more of an issue in general that types should be from Browser environment

Reply:
I've decided to remove it whatsoever and added a comment explaining why I don't want to risk any non-necessary types to appear in API